modules/openid_connect/app/services/openid_connect/user_tokens/fetch_service.rb

```diff
@@ -38,6 +38,10 @@ class FetchService
       include Dry::Monads[:result]
       include Dry::Monads::Do.for(:access_token_for, :refreshed_access_token_for)
 
+      TOKEN_OBTAINED = "access_token_obtained"
```

Comment:
:yellow_circle: I am wondering whether this should be defined on `OpenProject::Events`, because then it would read as `OpenProject::Events::TOKEN_OBTAINED` (like the other event we touched in this PR. Alternatively, I could imagine it being called `TOKEN_OBTAINED_EVENT`, which would in fully qualified form be called `OpenIDConnect::UserTokens::FetchService::TOKEN_OBTAINED_EVENT`.

Reply:
I avoided `OpenProject::Events`, because this event is produced from within the oidc module. Let's go with `TOKEN_OBTAINED_EVENT`.